### ISSUE TITLE
Shortcode: redirect_to attribute not working / removes query params 

### ISSUE THREAD

**Issue by marceldarvas** [rtCamp/login-with-google#159]
I've tested various formats (with or without a hostname) for redirect_to URL, but it always seems to redirect back to the URL that it was displayed on.

I found this bug to have been addressed before:
https://github.com/rtCamp/login-with-google/pull/96

I've checked my LiteSpeed Cache config + Cloudflare, but it remains an issue.

Additionally, I investigated this issue because from the Login Screen (with wp-login.php?redirect_to="") the query params are preserved and working fine.

There also seems to be a typo in the shortcode example [here](https://github.com/rtCamp/login-with-google/blob/0c4ee9c222ff6455b2e8c89d226e15708087401b/README.md?plain=1#L105) as there's a '/' at the end that doesn't belong there.

I'm not the only one seeming to have issues with this: https://github.com/rtCamp/login-with-google/issues/154

I've also tried using the rtcamp.google_default_redirect filter to override, but it didn't seem to work in my case.

I was wondering why do you have this code:
https://github.com/rtCamp/login-with-google/blame/0c4ee9c222ff6455b2e8c89d226e15708087401b/src/Modules/Shortcode.php#L147

**Comment by marceldarvas** [comment]
I have got the redirection to work. However, I am still experiencing issues caused by the shortcode.

I've found RtCamp\GoogleLogin\Modules\Shortcode::scan_shortcode() to be an issue when running the CSS Cache regeneration process via the Oxygen Builder Admin Panel.

`Argument #1 ($output) must be of type string, null given`

Upon disabling this plugin, the regeneration process runs fine. (I had errors in Oxygen Templates giving failed AJAX requests).
